autogen.sh

```diff
@@ -13,5 +21,5 @@ echo "----------------------------------------------------------------"
 echo "Initialized build system. For a common configuration please run:"
 echo "----------------------------------------------------------------"
 echo
-echo "./configure $args"
+echo "./configure HAVE_AS_KNOWS_AVX512=0 HAVE_AS_KNOWS_SHANI=0 AS=yasm $args"
```

Comment:
We certainly don't want to default to this.  Configure should check for availability and set the appropriate values for these.